main.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"mime"
+	"net"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/Percona-Lab/percona-version-service/server"
+	pbVersion "github.com/Percona-Lab/percona-version-service/versionpb"
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	"github.com/rakyll/statik/fs"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/grpclog"
+
+	_ "github.com/Percona-Lab/percona-version-service/statik"
+)
+
+func getOpenAPIHandler() http.Handler {
+	err := mime.AddExtensionType(".svg", "image/svg+xml")
+	if err != nil {
+		log.Fatalf("creating OpenAPI filesystem: %v", err)
+	}
+
+	statikFS, err := fs.New()
+	if err != nil {
+		log.Fatalf("creating OpenAPI filesystem: %v", err)
+	}
+
+	return http.FileServer(statikFS)
+}
+
+func main() {
+	useTLS := strings.ToLower(os.Getenv("SERVE_HTTP")) != "true"
```

Comment:
TLS will be terminated by a load balancer in SaaS. For local testing, you can use nginx, Traefik, or something and remove the TLS Go code to reduce complexity.

Or you want to terminate it there for non-SaaS setups without nginx/Traefik/something?

Reply:
>Or you want to terminate it there for non-SaaS setups without nginx/Traefik/something?
Yes, we discussed such possibility, that was my intention